packages/sources/coingecko/src/endpoint/price.ts

```diff
@@ -79,8 +65,23 @@ export const execute: ExecuteWithConfig<Config> = async (request, config) => {
   }
 
   const response = await Requester.request(options, customError)
+
+  if (Array.isArray(symbol)) {
```

Comment:
I would move this out to a named function

Reply:
Good suggestion. Changing.